### PR TITLE
Implement requestMtu on android

### PR DESCRIPTION
**Pull request by devxpy**
I didn't want to go through the whole process of creating a new protobuf, so just used normal json. I hope you don't mind :-)

Fixes https://github.com/pauldemarco/flutter_blue/issues/129

### CODE REVIEW

**Comment by bangonkali** [comment]
Thanks for this. Very important in my usecase. Hoping this will be merged soon. @pauldemarco 🤓

**Comment by devxpy** [comment]
@bangonkali If you're interested, I've built my own flutter BLE plugin - [rx_ble](https://pub.dev/packages/rx_ble).

**Comment by pauldemarco** [comment]
@devxpy 
Thanks for the contribution.

I will have to re-implement this using the protobuf messages on both Android and iOS side.

I'll leave this PR open for the time being.

File: pubspec.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: flutter_blue
 description: Bluetooth plugin for Flutter
-version: 0.5.0
+version: 0.6.0
```
Comment (andretietz) [inline pubspec.yaml]:
```suggestion
version: 0.5.0
```

Let @pauldemarco decide, to which version he want's to update it.

Comment (devxpy) [reply pubspec.yaml]: https://semver.org/

**Comment by pauldemarco** [comment]
Thanks @devxpy and @andretietz for your contribution.
I hope to have covered all the bases in PR #363 . Please try it out and let me know what you think.

File: pubspec.yaml (hunk 1 above)
Comment (andretietz) [reply pubspec.yaml]: > https://semver.org/

I just ment that if everyone would change the version in his PR, then there would be a hell of merge conflicts. This is why it is easier to let the author decide wether he's updating the version of the library or not. not every PR needs a version change.

Comment (devxpy) [reply pubspec.yaml]: Ah, ok

**Pull request closed without merging**